### PR TITLE
ISPN-7168 Prevent user from configuring passivation with a shared store

### PR DESCRIPTION
**Pull request by wburns**
https://issues.jboss.org/browse/ISPN-7168

### CODE REVIEW

File: documentation/src/main/asciidoc/upgrading/upgrading.asciidoc [hunk 1]
```diff
@@ -7,6 +7,17 @@ The Infinispan community
 
 This guide walks you through the process of upgrading Infinispan.
 
+== Upgrading from 9.3 to 9.4
+
+=== Store Changes
+
+==== Shared and Passivation
+
+A store cannot be configured as both shared and having passivation enabled.
```
Comment (anistor) [inline documentation/src/main/asciidoc/upgrading/upgrading.asciidoc]: Was just wondering, don't we also have a doc section for passivation where such fundamental limitations/implications of passivation should also be mentioned, besides the 'Upgrading' section?

Comment (wburns) [reply documentation/src/main/asciidoc/upgrading/upgrading.asciidoc]: 👍 let me make sure that is added.

**Comment by wburns** [comment]
@ryanemerson hehe, yeah I was hoping we didn't have things using it, but I will fix those.

**Comment by wburns** [comment]
This PR will not be making it into 9.4.0.Alpha1

**Comment by ryanemerson** [comment]
@wburns I think I'll remove the trivial label :stuck_out_tongue_winking_eye:

**Comment by wburns** [comment]
I just ran the entire server/integration/testsuite directory and found no failures. I hope CI can give me real results sometime soon so I can verify what actually has to be fixed.

**Comment by wburns** [comment]
A green build, such a rare sight to behold. https://ci.infinispan.org/job/Infinispan/job/PR-6142/7/

**Comment by anistor** [comment]
Indeed. We need a professional photographer to capture this! :)

**Pull request merged**